src/Ledger/TokenAlgebra.lagda

```diff
@@ -44,8 +40,6 @@ record TokenAlgebra : Type₁ where
 
   open MonoidMorphisms (rawMonoid) (Monoid.rawMonoid +-0-monoid) public
   field
-\end{code}
```

Comment:
I think this might still be needed to hide all the open stuff in between the fields